Sources/MapboxNavigation/Array.swift

```diff
@@ -52,6 +53,51 @@ extension Array where Iterator.Element == [CLLocationCoordinate2D]? {
 }
 
 extension Array where Iterator.Element == CLLocationCoordinate2D {
+
+    /**
+     Returns an array of congestion segments by associating the given congestion levels with the coordinates of the respective line segments that they apply to.
+     
+     This method coalesces consecutive line segments that have the same congestion level.
+     
+     For each item in the `CongestionSegment` collection a `CongestionLevel` substitution will take place that has a streets road class contained in the `roadClassesWithOverriddenCongestionLevels` collection.
+     For each of these items the `CongestionLevel` for `.unknown` traffic congestion will be replaced with the `.low` traffic congestion.
+     
+     - parameter congestionLevels: The congestion levels along a leg. There should be one fewer congestion levels than coordinates.
+     - parameter streetsRoadClasses: A collection of streets road classes for each geometry index in `Intersection`. There should be the same amount of `streetsRoadClasses` and `congestions`.
+     - parameter roadClassesWithOverriddenCongestionLevels: Streets road classes for which a `CongestionLevel` substitution should occur.
+     - returns: A list of `CongestionSegment` tuples with coordinate and congestion level.
+     */
+    func combined(_ congestionLevels: [CongestionLevel],
```

Comment:
Does this method use `self` at all? If not, it should either be static or use `self` instead of `congestionLevels`.

Reply:
Not sure it's possible in this case, this method is already applied only on arrays which contain `CLLocationCoordinate2D` objects and merges them with `CongestionLevel` objects.